api/src/main/java/net/kyori/adventure/text/Component.java

```diff
@@ -2216,6 +2223,33 @@ default void detectCycle(final @NotNull Component that) {
     return this;
   }
 
+  /**
+   * Sets the shadow color of this component.
+   *
+   * @param argb the color
+   * @return a component
+   * @since 4.18.0
+   */
+  @Contract(pure = true)
+  @Override
+  default @NotNull Component shadowColor(final @Nullable ARGBLike argb) {
```

Comment:
Is it worth mentioning here, or in the docs, that setting this to null doesn't _remove_ the shadow colour, it just restores it to default?